addons/hr_expense/__manifest__.py

```diff
@@ -30,6 +30,7 @@
     'data': [
         'security/hr_expense_security.xml',
         'security/ir.model.access.csv',
+        'data/digest_data.xml',
```

Comment:
Will require a migration script (in base, 13.4, pre-modules.py).

To propagate to other dependencies update, obviously